src/main/java/com/sickgyun/server/profile/presentation/ProfileController.java

```diff
@@ -0,0 +1,34 @@
+package com.sickgyun.server.profile.presentation;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.sickgyun.server.profile.presentation.dto.ProfileCreateRequestDto;
+import com.sickgyun.server.profile.service.ProfileCommandService;
+import com.sickgyun.server.user.domain.User;
+import com.sickgyun.server.user.service.UserTempService;
+
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("profiles")
+public class ProfileController {
```

Comment:
RequestMapping에 /를 빠트리신 거 같네요

```suggestion
@RestController
@RequiredArgsConstructor
@RequestMapping("/profiles")
public class ProfileController {
```

Reply:
ㅋㅋㅋㅋㅋㅋㅋ 넣을까 말까 고민을 했다가 웅빈님 저번에 안 넣는거 보고 뺐는데 넣기로 할까요??

Reply:
오잉? 저거 안 넣으면 localhost:8080profiles로 접근해야 되는 거 아닌가요?

Reply:
스프링에서 자동으로 / 넣어주더라구요